packages/api/.prettierrc

```diff
@@ -1,7 +1,7 @@
 
 {
   "singleQuote": true,
-  "printWidth": 100,
+  "printWidth": 150,
```

Comment:
this could be too long

Reply:
Actually if we keep it hundred the number of lines will be increased a lot, so I kept it 150.